Define _GNU_SOURCE to access CPU_ZERO() macro in uClibc

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: userspace-rcu-0.6.7/tests/urcutorture.c
===================================================================
--- userspace-rcu-0.6.7.orig/tests/urcutorture.c	2012-02-01 10:17:09.938804402 +0100
+++ userspace-rcu-0.6.7/tests/urcutorture.c	2012-02-01 10:17:15.308713372 +0100
@@ -1,3 +1,4 @@
+#define _GNU_SOURCE
 #include <string.h>
 #include <sys/time.h>
 #include <poll.h>
